subsys/bluetooth/mesh/pb_adv.c

```diff
@@ -46,7 +46,7 @@
 #define XACT_ID_MAX  0x7f
 #define XACT_ID_NVAL 0xff
 
-#define RETRANSMIT_TIMEOUT  K_MSEC(500)
+#define RETRANSMIT_TIMEOUT  K_MSEC(CONFIG_BT_MESH_PB_ADV_RETRANS_TIMEOUT)
```

Comment:
This looks like a bugfix for a separate issue. Could you put this in a separate commit? Can go in the same PR.

Reply:
ooh, i'm not see any bugfix for this? please point to me.

Reply:
I mean, this is fixing a separate issue from the transport.c segmentation timing, so it should be a separate commit